its/ruling/src/test/resources/autoscan/autoscan-diff-by-rules.json

```diff
@@ -2735,6 +2735,12 @@
     "falseNegatives": 0,
     "falsePositives": 0
   },
+  {
+    "ruleKey": "S6331",
+    "truePositives": 47,
+    "falseNegatives": 1,
```

Comment:
This FN is due to a regex present in an annotation. Without the semantic, we are not able to find it.
The is also the reason why we have to increase the "Issues differences" counter in "AutoScanTest"